src/modules/UI/scenes/TransactionList/TransactionList.ui.js

```diff
@@ -265,77 +179,71 @@ export class TransactionList extends Component<Props, State> {
-                        {fiatBalanceString}
+                    {!displayDenomination.symbol && (
+                      <T numberOfLines={1} style={styles.currentBalanceBoxBits}>
+                        {' ' + selectedCurrencyCode}
                       </T>
-                    </View>
-                  </View>
-                ) : (
-                  <View style={[styles.balanceHiddenContainer]}>
-                    <T style={[styles.balanceHiddenText]}>{SHOW_BALANCE_TEXT}</T>
-                  </View>
-                )}
-              </TouchableOpacity>
-              <View style={[styles.requestSendRow]}>
-                <TouchableHighlight style={[styles.requestBox, styles.button]} underlayColor={styleRaw.underlay.color} onPress={Actions.request}>
-                  <View style={[styles.requestWrap]}>
-                    <Image style={{ width: 25, height: 25 }} source={requestImage} />
-                    <T style={[styles.request]}>{REQUEST_TEXT}</T>
+                    )}
                   </View>
-                </TouchableHighlight>
-                <TouchableHighlight style={[styles.sendBox, styles.button]} underlayColor={styleRaw.underlay.color} onPress={Actions.scan}>
-                  <View style={[styles.sendWrap]}>
-                    <Image style={{ width: 25, height: 25 }} source={sendImage} />
-                    <T style={styles.send}>{SEND_TEXT}</T>
-                  </View>
-                </TouchableHighlight>
+                </View>
+                <View style={[styles.currentBalanceBoxDollarsWrap]}>
+                  <T numberOfLines={1} style={[styles.currentBalanceBoxDollars]}>
+                    {fiatBalanceString}
+                  </T>
+                </View>
               </View>
-            </Animated.View>
-          )}
+            )}
+            <View style={[styles.requestSendRow]}>
+              <TouchableHighlight style={[styles.requestBox, styles.button]} underlayColor={styleRaw.underlay.color} onPress={Actions.request}>
+                <View style={[styles.requestWrap]}>
+                  <Image style={{ width: 25, height: 25 }} source={requestImage} />
+                  <T style={[styles.request]}>{REQUEST_TEXT}</T>
+                </View>
+              </TouchableHighlight>
+              <TouchableHighlight style={[styles.sendBox, styles.button]} underlayColor={styleRaw.underlay.color} onPress={Actions.scan}>
+                <View style={[styles.sendWrap]}>
+                  <Image style={{ width: 25, height: 25 }} source={sendImage} />
+                  <T style={styles.send}>{SEND_TEXT}</T>
+                </View>
+              </TouchableHighlight>
+            </View>
+          </View>
         </Gradient>
-      </Animated.View>
+      </TouchableOpacity>
     )
   }
 
-  renderBalanceBoxTrue = this.renderBalanceBox(true)
-  renderBalanceBoxFalse = this.renderBalanceBox(false)
-  currentRenderBalanceBox = this.renderBalanceBoxTrue
-
```

Comment:
what were these things?

Reply:
In my opinion the previous implementation of the 'Show Balance' component (non-Redux) was "wonky" and this has been replaced with an implementation that is more straight-forward.